test/tcframe/runner/RunnerTests.cpp

```diff
@@ -137,6 +138,7 @@ TEST_F(RunnerTests, Run_Generation_UseConfigOptions) {
             .setMultipleTestCasesCounter(&T)
             .setSolutionCommand("./solution")
             .setOutputDir("tc")
+            .setGenerateOutput(true)
```

Comment:
This should be `false`; otherwise you're not testing anything.

In `ProblemSpecWithConfig` in this class, add `StyleConfig()` with `NoOutputFiles()` set.